metadata/routes.py

```diff
@@ -69,6 +69,24 @@ def get_credentials():
         response.status = e.status
         return {'error': {'message': e.message}}
 
+@route('/latest/dynamic/instance-identity/document')
+def get_identity_document():
+    return {
+        'privateIp' : '127.0.0.1',
+        'devpayProductCodes' : None,
+        'availabilityZone' : 'us-east-1a',
+        'version' : '2010-08-31',
+        'instanceId' : 'i-12345678',
+        'billingProducts' : None,
+        'instanceType' : 't2.nano',
```

Comment:
The instance type returned by the `/latest/meta-data/instance-type` route above is `r3.2xlarge`. I don't think it makes a difference what the returned instance type is, but it should be consistent. Add a constant to the top of the file (e.g. `INSTANCE_TYPE = 't2.nano'`) and use that in both places.